eng/sendToHelix.proj

```diff
@@ -16,36 +21,54 @@
     <FailOnMissionControlTestFailure>true</FailOnMissionControlTestFailure>
   </PropertyGroup>
 
-  <ItemGroup>
-    <!-- For now we only have a queue for Linux. Once we have a queue for Windows we will run 
-    those tests too. -->
+  <PropertyGroup Condition=" '$(HelixAccessToken)' == '' ">
+    <IsExternal>true</IsExternal>
```

Comment:
I think `IsExternal` is no longer needed. I think the Helix SDK will set it to true if `HelixAccessToken` is empty

Reply:
Ok, I'll check and if so I'll remove. Thanks